src/pages/image/_id.vue

```diff
@@ -31,12 +31,9 @@
       <VButton
         as="VLink"
         :href="image.foreign_landing_url"
```

Comment:
Nice cleanup! VLink default properties, and the redundant`keyup` handling for `click` events.